static/app/views/performance/browser/webVitals/components/performanceScoreRingWithTooltips.tsx

```diff
@@ -22,32 +30,116 @@ const {
   ttfb: TTFB_WEIGHT,
 } = PERFORMANCE_SCORE_WEIGHTS;
 
+type Coordinates = {
+  x: number;
+  y: number;
+};
+
+type WebVitalsLabelCoordinates = {
+  [p in WebVitals]?: Coordinates;
+};
+
 type Props = {
   height: number;
   projectScore: ProjectScore;
   ringBackgroundColors: string[];
   ringSegmentColors: string[];
   text: React.ReactNode;
   width: number;
+  barWidth?: number;
   hideWebVitalLabels?: boolean;
+  inPerformanceWidget?: boolean;
+  projectData?: TableData;
   size?: number;
+  webVitalLabelCoordinates?: WebVitalsLabelCoordinates;
   x?: number;
   y?: number;
 };
 
+type WebVitalLabelProps = {
+  coordinates: Coordinates;
+  inPerformanceWidget: boolean;
+  location: Location;
+  onHover: (webVital: WebVitals) => void;
+  onUnHover: () => void;
+  organization: Organization;
+  webVital: WebVitals;
+  projectData?: TableData;
+  webVitalLabelCoordinates?: WebVitalsLabelCoordinates;
+};
+
+function WebVitalLabel({
+  organization,
+  location,
+  webVital,
+  coordinates,
+  onHover,
+  onUnHover,
+  webVitalLabelCoordinates,
+  inPerformanceWidget,
+  projectData,
+}: WebVitalLabelProps) {
+  const xOffset = webVitalLabelCoordinates?.[webVital]?.x ?? 0;
+  const yOffset = webVitalLabelCoordinates?.[webVital]?.y ?? 0;
+  const webvitalInfo =
+    webVital === 'cls'
+      ? Math.round((projectData?.data?.[0]?.['p75(measurements.cls)'] as number) * 100) /
+        100
+      : getFormattedDuration(
+          (projectData?.data?.[0]?.[`p75(measurements.${webVital})`] as number) / 1000
+        );
+
+  return (
+    <Link
+      to={{
+        pathname: `/organizations/${organization.slug}/performance/browser/pageloads/`,
+        query: {
+          ...location.query,
+          webVital,
+        },
+      }}
+      disabled={!inPerformanceWidget}
+    >
+      <ProgressRingText
```

Comment:
I think this only fires the on hover effect when mousing over the Webvital label ie the LCP part. The hover region should also include the subtext since we're using a mouse pointer there too.

Reply:
Yep will fix this.